### PR TITLE
Add alternative way for current_layer

### PR DESCRIPTION
**Pull request by nelsongraca**
Get current layer information from into inside print_stats if it exists

### CODE REVIEW

**Review by marcolivierarsenault — COMMENTED** [review]
This makes sense. 

Couple of comments:
- Please add some testing for this scenario
- Add documentation for this alternate way of getting doc
- Make sure your branch is up to date before merging
- CI is failing make sure to run the pre-commit hook (should be available in VSCode tasks)

**Comment by nelsongraca** [comment]
As soon as I have time I'll take care of it, this was a quick commit done
on GitHub directly.

**Comment by marcolivierarsenault** [comment]
Perfect thanks for the help

**Comment by nelsongraca** [comment]
@marcolivierarsenault documentation missing planning on adding today

**Comment by nelsongraca** [comment]
@marcolivierarsenault docs added please review.

File: docs/entities/sensors.rst [hunk 1]
```diff
@@ -136,6 +136,13 @@ Binary Sensors are used to represent a single binary value. They can are used fo
     - True if filament is missing
     - From Moonraker API
 
+Curremt Layer
```
Comment (marcolivierarsenault) [inline docs/entities/sensors.rst:139]: Neat

**Review by marcolivierarsenault — APPROVED** [review]

File: docs/entities/sensors.rst (hunk 1 above)
Comment (marcolivierarsenault) [reply docs/entities/sensors.rst:139]: Ooo just saw a typo, I will fix

Comment (marcolivierarsenault) [inline docs/entities/sensors.rst:139]:
```suggestion
Current Layer
```

**Pull request merged**